samples/test/translate_model_management_test.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {AutoMlClient} = require('@google-cloud/automl');
+
+const cp = require('child_process');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+// const DATASET_ID = 'TRL8522556519449886720';
+const LIST_MODEL_REGION_TAG = 'list_models';
+const GET_MODEL_REGION_TAG = 'get_model';
+const LIST_MODEL_EVALUATION_REGION_TAG = 'list_model_evaluations';
+const GET_MODEL_EVALUATION_REGION_TAG = 'get_model_evaluation';
+const LIST_OPERATION_STATUS_REGION_TAG = 'list_operation_status';
+const GET_OPERATION_STATUS_REGION_TAG = 'get_operation_status';
+
+// const CREATE_MODEL_REGION_TAG = 'translate_create_model';
+// const DELETE_MODEL_REGION_TAG = 'delete_model';
+
+describe('Automl Translate Model Tests', () => {
+  const client = new AutoMlClient();
+
+  it('should list and get model, list and get model evaluations', async () => {
```

Comment:
i think like create a dataset, this needs to be split up into three tests. I know model creation takes forever though, so it possibly should be mocked in the test